backend/controller/ingress/testdata/go/httpingress/httpingress.go

```diff
@@ -103,62 +102,60 @@ type QueryParamRequest struct {
 }
 
 //ftl:ingress http GET /queryparams
-func Query(ctx context.Context, req builtin.HttpRequest[QueryParamRequest]) (builtin.HttpResponse[string, string], error) {
+func Query(ctx context.Context, req builtin.HttpRequest[ftl.Unit, ftl.Unit, QueryParamRequest]) (builtin.HttpResponse[string, string], error) {
 	return builtin.HttpResponse[string, string]{
-		Body: ftl.Some(req.Body.Foo.Default("No value")),
+		Body: ftl.Some(req.Query.Foo.Default("No value")),
 	}, nil
 }
 
-type HtmlRequest struct{}
-
 //ftl:ingress http GET /html
-func Html(ctx context.Context, req builtin.HttpRequest[HtmlRequest]) (builtin.HttpResponse[string, string], error) {
+func Html(ctx context.Context, req builtin.HttpRequest[ftl.Unit, ftl.Unit, ftl.Unit]) (builtin.HttpResponse[string, string], error) {
 	return builtin.HttpResponse[string, string]{
 		Headers: map[string][]string{"Content-Type": {"text/html; charset=utf-8"}},
 		Body:    ftl.Some("<html><body><h1>HTML Page From FTL 🚀!</h1></body></html>"),
 	}, nil
 }
 
 //ftl:ingress http POST /bytes
-func Bytes(ctx context.Context, req builtin.HttpRequest[[]byte]) (builtin.HttpResponse[[]byte, string], error) {
+func Bytes(ctx context.Context, req builtin.HttpRequest[[]byte, ftl.Unit, ftl.Unit]) (builtin.HttpResponse[[]byte, string], error) {
 	return builtin.HttpResponse[[]byte, string]{Body: ftl.Some(req.Body)}, nil
 }
 
 //ftl:ingress http GET /empty
-func Empty(ctx context.Context, req builtin.HttpRequest[ftl.Unit]) (builtin.HttpResponse[ftl.Unit, string], error) {
+func Empty(ctx context.Context, req builtin.HttpRequest[ftl.Unit, ftl.Unit, ftl.Unit]) (builtin.HttpResponse[ftl.Unit, string], error) {
 	return builtin.HttpResponse[ftl.Unit, string]{Body: ftl.Some(ftl.Unit{})}, nil
 }
 
-//ftl:ingress http GET /string
-func String(ctx context.Context, req builtin.HttpRequest[string]) (builtin.HttpResponse[string, string], error) {
+//ftl:ingress http POST /string
```

Comment:
Why did these change to `POST`?

Reply:
They have a body, and GET requests with a body don't make sense.